01_science_technology_and_epistemology/01_science_technology_and_epistemology.jl

```diff
@@ -8,72 +8,186 @@ using InteractiveUtils
+With technology we transform our experience in the world.
 
-The current understanding of how knowledge is produced is very strange and even counterproductive for the development of humanity. Nowadays, there is a kind of *linear thinking* model in which we take for granted that the process of knowledge production is very definite: We first make academic and theoretical discoveries and only then, with all of that formalization made, we can start thinking of putting it to practical use in the real world. This is a completely wrong way of thinking and is holding us back enormously, causing important funds to be misused.
+It is evident that the more we know about the reality in which we live, the more we will be able to modify it. 
+But from this idea comes a very important question: how is the "order" in which innovation takes place? 
+Is it necessary first to have an absolute understanding of the process we are trying to modify? 
+Or is technology created in a more chaotic process, through trial and error?
 
+So let´s think for a moment, does a child need to know the gear mechanisms of a bicycle, in order to be able to use it? does Messi need to know about fluid dynamics to make the ball take a curved trajectory? Did the Romans need to know the Navier-Stokes equation in order to build their huge aqueducts? 
+Maybe you are thinking that these are very specific cases, but the reality is that those example share a general and universal property: The knowledge is acquired based on experimentation, implementation and heuristics, pure trial and error. 
+And technological innovation follows the same mechanism, it was discovered by people that were  taking risks, taking action, with a pure trial-and-error experiential manner and, above all, without anyone truly understanding the theory behind.
 
-If you start looking at the technology/innovation history you may gradually begin to notice a repeating pattern: There is a constant differentiation between the people who *do* and the people who *watch* and then narrate. "The History Written by the Losers" would [Nassim Nicholas Taleb](https://en.wikipedia.org/wiki/Nassim_Nicholas_Taleb) said, the people that are doing stuff don't have time for beauty writing. But the thing does not stop there. It is not only that the do-nothings who write about the findings of others, but also attribute it to them! And this comes from a worrying bias that has been developing throughout recent history: the linear thinking we are talking about. That –apparently– common sense in which knowledge is built from a purely intellectual work that can be done in the armchair at home, and that after acquiring this sacred theoretical knowledge it is possible to come up with technology or innovation.
+The current understanding of how knowledge is produced is quite out of step with what really happen, and may even work against us in our development. 
```

Comment:
The general idea is correct, but I would work on  this sentence a bit more. 
- "happen" is gramatically incorrect here.
- Who is "us" (students in general? people trying to learn technology?)
- Try to avoid using "we", "our", "us" and use more specific subjects